src/styles/theme.config.tsx

```diff
@@ -41,6 +41,7 @@ export class THEMES {
       theme_dark_green: "#22b9a5",
       theme_light_green: "#86eace",
       theme_light_blue: "#4fc2f8",
+      theme_lighter_blue: "#f5faff",
```

Comment:
Ah shoot its too light, lets try this
```suggestion
      theme_lighter_blue: "#F8FAFD",
```